codec_specs.md

```diff
@@ -157,276 +220,466 @@ A `Codec Mapping` MAY only be defined with a `Superseded By` value, if it has an
+Initialisation: none
+
+### A_AC3/BSID10
+
+Codec ID: A_AC3/BSID10
+
+Codec Name: (Dolby™) AC3
+
+Description: The ac3 frame header has, similar to the mpeg-audio header a version field. Normal ac3 is defiened as bitstream id 8 (5 Bits, numbers are 0-15). Everything below 8 is still compatible with all decoders that handle 8 correctly. Everything higher are additions that break decoder compatibility.
+For the samplerates 24kHz (00); 22,05kHz (01) and 16kHz (10) the BSID is 9
+For the samplerates 12kHz (00); 11,025kHz (01) and 8kHz (10) the BSID is 10
+
+Initialisation: none
+
+### A_ALAC
+
+Codec ID: A_ALAC
+
+Codec Name: ALAC (Apple Lossless Audio Codec)
+
+Initialisation: The `Private Data` contains ALAC's magic cookie (both the codec specific configuration as well as the optional channel layout information). Its format is described in [ALAC's official source code](http://alac.macosforge.org/trac/browser/trunk/ALACMagicCookieDescription.txt).
+
+### A_DTS
+
+Codec ID: A_DTS
+
+Codec Name: Digital Theatre System
+
+Description: Supports DTS, DTS-ES, DTS-96/26, DTS-HD High Resolution Audio and DTS-HD Master Audio. The private data is void. Corresponding ACM wFormatTag : 0x2001
+
+Initialisation: none
+
+### A_DTS/EXPRESS
+
+Codec ID: A_DTS/EXPRESS
+
+Codec Name: Digital Theatre System Express
+
+Description: DTS Express (a.k.a. LBR) audio streams. The private data is void. Corresponding ACM wFormatTag : 0x2001
+
+Initialisation: none
+
+### A_DTS/LOSSLESS
+
+Codec ID: A_DTS/LOSSLESS
+
+Codec Name: Digital Theatre System Lossless
+
+Description: DTS Lossless audio that does not have a core substream. The private data is void. Corresponding ACM wFormatTag : 0x2001
+
+Initialisation: none
+
+### A_VORBIS
+
+Codec ID: A_VORBIS
+
+Codec Name: Vorbis
+
+Initialisation: The `Private Data` contains the first three Vorbis packet in order. The lengths of the packets precedes them. The actual layout is:
+- Byte 1: number of distinct packets '`#p`' minus one inside the CodecPrivate block. This MUST be '2' for current (as of 2016-07-08) Vorbis headers.
+- Bytes 2..n: lengths of the first '`#p`' packets, coded in [Xiph-style lacing]({{site.baseurl}}/index.html#lacing). The length of the last packet is the length of the CodecPrivate block minus the lengths coded in these bytes minus one.
+- Bytes n+1..: The [Vorbis identification header](http://www.xiph.org/ogg/vorbis/doc/vorbis-spec-ref.html), followed by the [Vorbis comment header](http://www.xiph.org/ogg/vorbis/doc/v-comment.html) followed by the [codec setup header](http://www.xiph.org/ogg/vorbis/doc/vorbis-spec-ref.html).
```

Comment:
Updated to https://xiph.org/vorbis/doc/Vorbis_I_spec.html and https://xiph.org/vorbis/doc/v-comment.html.